### DISCUSSION TITLE
Run npm audit signatures

### DISCUSSION THREAD

**Discussion by vitalik203** [semantic-release/semantic-release#3238] [Commit Convention]
npm audit signatures
shell: /usr/bin/bash -e {0}
audited 552 packages in 4s

545 packages have verified registry signatures

55 packages have verified attestations

7 packages have invalid attestations:

@semantic-release/commit-analyzer@12.0.0 (https://registry.npmjs.org/)
@semantic-release/github@10.0.2 (https://registry.npmjs.org/)
@semantic-release/npm@12.0.0 (https://registry.npmjs.org/)
axios@1.6.8 (https://registry.npmjs.org/)
playwright@1.42.1 (https://registry.npmjs.org/)
playwright-core@1.42.1 (https://registry.npmjs.org/)
semantic-release@23.0.5 (https://registry.npmjs.org/)

Someone might have tampered with these packages since they were published on the registry!

Process completed with exit code 1.

**Comment by travi** [comment]
See context in https://github.com/semantic-release/npm/issues/763#issuecomment-1989542327

tl;dr update your npm cli to the latest version
